renku/ui/cli/project.py

```diff
@@ -55,6 +55,9 @@
 | -m, --metadata    | Path to json file containing custom metadata to be   |
 |                   | added to the project knowledge graph.                |
 +-------------------+------------------------------------------------------+
+| -u, --unset <type>| Remove a value from a project. One of ``creators``,  |
```

Comment:
`unset` shouldn't accept creators (it doesn't in the code above). Since we need a creator for a dataset and we shouldn't allow unsetting it.